context/app/static/js/shared-styles/Links/ContactUsLink.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ComponentProps, MouseEvent, PropsWithChildren } from 'react';
+import { trackLink } from 'js/helpers/trackers';
+import IconLink from './iconLinks/IconLink';
+import { EmailIcon } from '../icons';
+
+type ContactUsLinkProps = PropsWithChildren<Omit<ComponentProps<typeof IconLink>, 'isOutbound' | 'href'>> & {
+  iconFontSize?: string;
+};
+
+const href = 'https://hubmapconsortium.org/contact-form/';
+const defaultText = 'contact us';
+
+function getHelpEvent() {
+  trackLink(window.location.href, 'help_form');
+}
+
+function ContactUsLink({ iconFontSize, children, ...props }: ContactUsLinkProps) {
+  return (
+    <IconLink
+      href={href}
+      isOutbound
+      iconOnLeft={false}
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      icon={<EmailIcon $fontSize={iconFontSize} />}
```

Comment:
Should we continue to use the e-mail icon? If not, should we just use `OutboundIconLink`?

Reply:
@tsliaw ?

Reply:
Maybe not the outbound icon since we don't use the outbound icon for other "within" HuBMAP links. How about the Material "[ContactSupport](https://fonts.google.com/icons?selected=Material%20Symbols%20Rounded%3Acontact_support%3AFILL%400%3Bwght%40400%3BGRAD%400%3Bopsz%4024)" icon"? 
Feel free to disagree though since I'm not sure either. If not, let's keep the email icon for now, even if it's misleading, since users should be used to that icon for help/contact forms.
![Screenshot 2023-09-07 at 12 52 33 PM](https://github.com/hubmapconsortium/portal-ui/assets/16737715/b8dce70f-2f47-42b2-ac23-c367bcd5c9b8)

Reply:
I've swapped out the EmailIcon for the ContactSupportOutlined Icon:

Sample usage:

![image](https://github.com/hubmapconsortium/portal-ui/assets/19957804/280f4a37-d83f-4f72-9af8-60d691f6f253)
![image](https://github.com/hubmapconsortium/portal-ui/assets/19957804/6fcf0c73-7b7a-4d9c-a97f-44de06e8c0f6)
![image](https://github.com/hubmapconsortium/portal-ui/assets/19957804/a6e2bc72-a4c6-4f0d-8784-158b4462b78e)